timestrap/static_src/tags/app.vue

```diff
@@ -24,7 +24,7 @@
                         </router-link>
                     </li>
                     <li class="nav-item">
-                        <router-link class="nav-link" :to="tasks">
+                        <router-link class="nav-link" :to="tasks" v-if="global.perms.view_task">
```

Comment:
A bit of a hiccup here - because the permissions promise usually doesn't finish before rendering `app`, the menu item does not appear even if the user does have the `view_task` permission. Unlike in the other vue tags, this does not update automatically once the promise completes (and updates `global`). But it does work after loading another vue (e.g. clicking the "Clients" router link). Not quite sure why this doesn't work fly...